### ISSUE TITLE
SD card test issues?

### ISSUE THREAD

**Issue by andypiper** [paperdink/paperd_ink_test#2]
I've had the SD card test fail in two different ways:

- if I use a 32Gb SD card, I'm seeing just an "SD card failed" error (even after reformatting as FAT32 and putting the image.bmp file on there)
- if I use an 8Gb card formatted in the same way, it gets to the SD card test and seems to load the bitmap (seen in the serial console), but it doesn't seem to be able to display it correctly - the screen has a black rectangle during the test.

**Comment by rgujju** [comment]
It should work now.

**Comment by andypiper** [comment]
Thanks.

Updated test results:

- 32Gb SD card continues to fail even after a reformat and copying the new BMP file. This was a PNY brand card, FWIW (the 8Gb card was a SanDisk)
- tried a 2Gb Verbatim card (an old one, looks like I probably hadn't used it in ~10 years)... also failed
- tried a 16Gb SanDisk with the new image -> works

tl;dr could be a combination of card quality, formatting, and also the image file (which now seems to work just fine).

Thanks for taking a look here!
